kotlinx-coroutines-core/common/src/CancellableContinuation.kt

```diff
@@ -227,17 +227,20 @@ public suspend inline fun <T> suspendAtomicCancellableCoroutine(
  *  Suspends coroutine similar to [suspendAtomicCancellableCoroutine], but an instance of [CancellableContinuationImpl] is reused if possible.
  */
 internal suspend inline fun <T> suspendAtomicCancellableCoroutineReusable(
```

Comment:
This method is no longer "atomic" as its name suggests

Reply:
Fixed